client/src/components/game/lab2/components/Repair.js

```diff
@@ -0,0 +1,579 @@
+        };
+    
+        //Closes the background color change popup
+        const closeBackground = () => {
+          this.onControlBackgroundPopup(false);
+        };
+    
+        ///Revers the background color to the default color
+        const revertBackground = () => {
+          this.setState({ background: this.state.background });
+          this.onControlBackgroundPopup(false);
+        };
+    
+        //Opens the correct circle color change popup
+        const changeCorrectColor = () => {
+          this.onControlCorrectPopup(true);
+        };
+    
+        //Closes the correct circle color change popup
+        const closeCorrectColor = () => {
+          this.onControlCorrectPopup(false);
+        };
+    
+        ///Revers the correct circle color to the default color
+        const revertCorrectColor = () => {
+          this.setState({ correctColor: this.state.correctCircle });
+          this.onControlCorrectPopup(false);
+        };
+    
+        //Opens the first incorrect circle color change popup
+        const changeIncorrectColorOne = () => {
+          this.onControlIncorrectPopupOne(true);
+        };
+    
+        //Closes the first incorrect color change popup
+        const closeIncorrectColorOne = () => {
+          this.onControlIncorrectPopupOne(false);
+        };
+    
+        ///Revers the first incorrect circle color to the default color
+        const revertIncorrectColorOne = () => {
+          this.setState({ incorrectColorOne: this.state.incorrectColorOne });
+          this.onControlIncorrectPopupOne(false);
+        };
+    
+        //Opens the second incorrect circle color change popup
+        const changeIncorrectColorTwo = () => {
+          this.onControlIncorrectPopupTwo(true);
+        };
+    
+        //Closes the second incorrect color change popup
+        const closeIncorrectColorTwo = () => {
+          this.onControlIncorrectPopupTwo(false);
+        };
+    
+        ///Revers the second incorrect circle color to the default color
+        const revertIncorrectColorTwo = () => {
+          this.setState({ incorrectColorTwo: this.state.incorrectColorTwo });
+          this.onControlIncorrectPopupTwo(false);
+        };
+    
+        if (this.state.background === "") {
+          this.setState({
+            background: this.props.colors[0],
+            correctColor: this.props.colors[1],
+            incorrectColorOne: this.props.colors[2],
+            incorrectColorTwo: this.props.colors[3]
+          });
+        }
+    const cssFileClasses = classNames({
+			code_editor__file: true,
+			'code_editor__file--active': true
+		});
+
+		const cssFileCodeClasses = classNames({
+			code_editor__code: true,
+			code_editor__hide: false
+		});
+
+		return (
+        <div className="code_editor">
+          
+          <div className="code_editor__content">
+            <div className={cssFileClasses}>
+              <div className='code_editor__file--active'>
```

Comment:
It looks like Circles.css has two borders around it?
<img width="142" alt="Screen Shot 2021-03-08 at 5 24 00 PM" src="https://user-images.githubusercontent.com/54957497/110389844-1a6d7300-8033-11eb-93bf-b3d1571222fa.png">

Reply:
I addressed it It still has a border but looks nicer. I'm not sure why it doesn't follow the other pages its the same if im not mistaken.